ui/apps/platform/cypress/constants/apiEndpoints.js

```diff
@@ -50,11 +50,7 @@ export const alerts = {
 };
 
 export const clusters = {
-    single: 'v1/clusters/**',
     list: 'v1/clusters',
```

Comment:
Did you mean to leave the `list` member of this object?

Reply:
Good eyes. 3 occurrences of `api.clusters.list` in helpers or tests for other containers.